tests/unit/test_useragent.py

```diff
@@ -0,0 +1,59 @@
+import contextlib
+import functools
+import typing
+from http.server import BaseHTTPRequestHandler
+
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.lsql.__about__ import __version__
+from databricks.labs.lsql.dashboards import Dashboards
+
+
+@contextlib.contextmanager
+def http_fixture_server(handler: typing.Callable[[BaseHTTPRequestHandler], None]):
+    from http.server import HTTPServer
+    from threading import Thread
+
+    class _handler(BaseHTTPRequestHandler):
+        def __init__(self, handler: typing.Callable[[BaseHTTPRequestHandler], None], *args):
+            self._handler = handler
```

Comment:
Is this on-purpose before the `supre().__init__`? Does that expect a handler to be present?